packages/webapp/pages/plus/index.tsx

```diff
@@ -17,21 +24,69 @@ const PlusDesktop = dynamic(() =>
   ).then((mod) => mod.PlusDesktop),
 );
 
-const PlusPage = (): ReactElement => {
+type PlusPageProps = Pick<GiftUserContextData, 'giftToUser'>;
+
+const seo: NextSeoProps = {
+  title: getTemplatedTitle('Unlock Premium Developer Features with Plus'),
+  openGraph: { ...defaultOpenGraph },
+  description:
+    'Upgrade to daily.dev Plus for an ad-free experience, custom feeds, bookmark folders, clickbait shield, and more.',
+};
+
+const PlusPage = ({ giftToUser }: PlusPageProps): ReactElement => {
+  const [userToGift, setUserToGift] = useState(giftToUser);
   const { isReady } = useRouter();
   const isLaptop = useViewSize(ViewSize.Laptop);
+  const pageProps: GiftUserContextData = {
+    giftToUser: userToGift,
+    onUserChange: setUserToGift,
```

Comment:
should we manage this in query params, it would then also work on page reload keeping the last user?